src/components/Home/Home.js

```diff
@@ -157,6 +157,11 @@ class Home extends Component {
             <div className={events}>
               <Link to="events">Voir nos prochains évènements</Link>
             </div>
+            
+            <h2 id="evenements">Événements passé</h2>
+            <div className={events}>
+              <Link to="events">Voir nos anciens évènements</Link>
```

Comment:
Should it actually link to the same page?
Instead of having twice the same link saying something different, why not just offering from here to view all events?